suites/nautilus/dashboard/sanity_containerized_ceph_dashboard.yaml

```diff
@@ -0,0 +1,104 @@
+tests:
+   - test:
+      name: install ceph pre-requisites
+      module: install_prereq.py
+      abort-on-fail: true
+
+   - test:
+        name: containerized ceph with dashboard
+        polarion-id: CEPH-83573534
+        module: test_ansible.py
+        config:
+          is_mixed_lvm_configs: True
+          ansi_config:
+              ceph_test: True
+              ceph_origin: distro
+              ceph_stable_release: nautilus
+              ceph_repository: rhcs
+              osd_scenario: lvm
+              osd_auto_discovery: False
+              journal_size: 1024
```

Comment:
remove journal_size if its not needed in lvm bluestore osd cases

Reply:
@XDsubhash yes removed journal_size as it's not needed